packages/primitives/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "src",
+		"declarationDir": "build-types",
+
+		"noImplicitAny": false,
+		"strictNullChecks": false
```

Comment:
See how far you can get without disabling these strictness checks. #21248 would have required a lot of changes to finish, I'd hope to enable them for @wordpress/element in the future.